lib/rules/value-keyword-case/README.md

```diff
@@ -164,3 +164,52 @@ a {
 }
 ```
 
+### `ignoreProperties: ["/regex/", "non-regex"]`
+
+Ignore case of the values of the listed properties.
+
+For example, with `"lower"`.
+
+```js
```

Comment:
Please include some regular expression examples [we do here](https://github.com/Bilie/stylelint/tree/140f09771b9eb26538537e75469a850313ea617b/lib/rules/value-keyword-case#ignorekeywords-regex-non-regex) for the `ignoreKeywords` option.

Also, can you please move the docs for this option above the `ignoreKeywords` option, as options should be listed alphabetically.

Reply:
Hi, thank you very much for your feedback! I was not sure if the new option should be called ignoreProperties or ignoreCustomProperties. Currently it is named ignoreProperties. Should it still go above ignoreKeywords in the readme or should I rename it to ignoreCustomProperties? Thank you!

Reply:
It should be called `ignoreProperties`. _Custom properties_ are [something different entirely](https://www.w3.org/TR/css-variables-1/#defining-variables) in CSS. There might come a time when a user requests an `ignoreCustomProperties` option but that would be for the following example:

```js
"value-keyword-case": ["lower", { ignoreCustomProperties: ["brand-color"] }]
```

```css
:root {
  --brand-color: myMadeUPKeYwOrDColor;
}
```

However, for what you're trying to achieve in this PR, `ignoreProperties` is the correct name.

Reply:
> Also, can you please move the docs for this option above the ignoreKeywords option, as options should be listed alphabetically.

Whoops, I just realised my mistake here... `K` is before `P` in the alphabet... silly me. I can see why you were asking about custom properties now.